examples/cgroup-rstat-flushing.bpf.c

```diff
@@ -0,0 +1,139 @@
+/*
+ * Measure cgroup rstat (recursive stats) flushing overhead and latency.
+ *
+ * Loosely based on bpftrace script cgroup_rstat_tracepoint.bt
+ *  - https://github.com/xdp-project/xdp-project/blob/master/areas/latency/cgroup_rstat_tracepoint.bt
+ *
+ * Depends on tracepoints added in kernel v6.10
+ */
+
+#include <vmlinux.h>
+#include <bpf/bpf_tracing.h>
+#include "maps.bpf.h"
+
+#define MAX_CGRP_LEVELS	5
+
+struct {
+	__uint(type, BPF_MAP_TYPE_PERCPU_ARRAY);
```

Comment:
The tabs are making clang-format unhappy in CI.

Reply:
IMHO then BPF  restricted-C code should be written with same tabs style as kernel code.
Lets change for formatting requirements, please!

Reply:
Let's leave it for another issue. For now let's just make clang-format happy.

Reply:
This projects `.clang-format` file states:
`# Minimal format loosely matching the kernel style.`
IMHO it doesn't match kernel style at all...

As it contains:
```
IndentWidth: 4
TabWidth: 4
UseTab: Never
```
Looking at kernel source in `.clang-format` this should have been:
```
IndentWidth: 8
TabWidth: 8
UseTab: Always
```

I will respect that this project requires its own special clang-format rule and obey to these, but I still think they are wrong.